clients/gtest/known_bugs.yaml

```diff
@@ -5,11 +5,12 @@
 #- { function: a_type: bf16_r, b_type: bf16_r, c_type: bf16_r, d_type: bf16_r, compute_type: f32_r, transA: N, transB: N, M: 512, N: 512, K: 512, lda: 512, ldb: 512, ldc: 512, ldd: 512, alpha: 5.0, alphai: 0.0, beta: 0.0, betai: 0.0, known_bug_platforms: gfx908 }
 
 Known bugs:
-- { function: matmul, grouped_gemm: 1, known_bug_platforms: "gfx1100, gfx1101, gfx1102" }
-- { function: matmul, grouped_gemm: 2, known_bug_platforms: "gfx1100, gfx1101, gfx1102" }
-- { function: matmul, grouped_gemm: 5, known_bug_platforms: "gfx1100, gfx1101, gfx1102" }
-- { function: matmul, grouped_gemm: 7, known_bug_platforms: "gfx1100, gfx1101, gfx1102" }
-- { function: matmul, gradient: 1, known_bug_platforms: "gfx1100, gfx1101, gfx1102" }
+- { function: matmul, grouped_gemm: 1, known_bug_platforms: "gfx1100, gfx1101, gfx1102, gfx1200, gfx1201" }
+- { function: matmul, grouped_gemm: 2, known_bug_platforms: "gfx1100, gfx1101, gfx1102, gfx1200, gfx1201" }
+- { function: matmul, grouped_gemm: 5, known_bug_platforms: "gfx1100, gfx1101, gfx1102, gfx1200, gfx1201" }
+- { function: matmul, grouped_gemm: 7, known_bug_platforms: "gfx1100, gfx1101, gfx1102, gfx1200, gfx1201" }
+- { function: matmul, gradient: 1, known_bug_platforms: "gfx1100, gfx1101, gfx1102, gfx1200, gfx1201" }
```

Comment:
when can we remove the workaround from testing_matmul.hpp
        //workaround before known_bug work
        if((gpu_arch_match(deviceProperties.gcnArchName, "11?") || gpu_arch_match(deviceProperties.gcnArchName, "12?"))
           && (arg.gradient || arg.grouped_gemm)) 
        {
            hipblaslt_cerr << "No Solution Found!!" << std::endl;
            return;
        }